docker-compose.yml

```diff
@@ -42,3 +38,7 @@ services:
 networks:
   backend:
       driver: bridge
+
+volumes:
+  mongo-data:
+  report-templates:
```

Comment:
Are you sure about the changes in this file?
Broke everything for me...

Reply:
Thanks for your check up!

You can fix this by copying the previous `mongo-data` and `report-templates` folder into the corresponding volume on the system.

I think the current file handling is not optimal either with putting the user-data into the code directories.

By using docker volumes on the other hand, we can archive the following:
- easy moving the code folder to another place within the filesystem (currently that is not possible without `sudo` / `root` permissions) and keeping the files
- better compatibility on different operating systems
- compliance with the docker guide on [Run your app in production](https://docs.docker.com/storage/volumes/)
> Volumes are the preferred mechanism for persisting data generated by and used by Docker containers. While bind mounts are dependent on the directory structure and OS of the host machine, volumes are completely managed by Docker. Volumes have several advantages over bind mounts:
>
>    - Volumes are easier to back up or migrate than bind mounts.
>    - You can manage volumes using Docker CLI commands or the Docker API.
>    - Volumes work on both Linux and Windows containers.
>    - Volumes can be more safely shared among multiple containers.
>    - Volume drivers let you store volumes on remote hosts or cloud providers, to encrypt the contents of volumes, or to add other functionality.
>    - New volumes can have their content pre-populated by a container.
>    - Volumes on Docker Desktop have much higher performance than bind mounts from Mac and Windows hosts.


But you're right, this should be included in the release notes when getting merged.

---
@yeln4ts do you plan to introduce version numbering and a release-system, so users can see what the latest fixes are and if there will be some breaking changes in the new version instead of just pulling all changes from `master`?

Reply:
I'm thinking about version numbering and changelog but I have to take the time to read about it. I think about introducing it when this is stable enough (so not too far from now).
Until then I don't want to introduce breaking changes if possible.

Reply:
Yes I would like to introduce some kind of release system with changelogs when I have something stable enough so that I can start with a clean slate (should be soon now). 
Until then, I prefer not introducing breaking changes if possible.